facebook/hhvm
# Add BUILD_ALWAYS ON to first-party dependencies

According to [the document](https://cmake.org/cmake/help/latest/module/ExternalProject.html), `BUILD_ALWAYS ON`BUILD_ALWAYS ON is recommended for external projects whose source code is not downloaded, otherwise the external projects will not be built when updating the source code.

## Test Plan:
1. Build HHVM in a GitHub Codespace for this Pull Request
2. `git pull` to includes some other commits where some source files in first-party dependences are changed.
3. Run `make` again

The changed source files should be rebuilt with the `BUILD_ALWAYS ON` option.
